Persistence/Repositories/BankAccountRepository.cs

```diff
@@ -41,3 +40,12 @@ public async Task SaveAsync(CancellationToken ct)
         await _context.SaveChangesAsync(ct);
     }
 }
+
+public interface IBankAccountRepository
```

Comment:
This interface should remain in the Domain project. See https://stackoverflow.com/a/72399927 for reasons why